feature_tests/features/producer/producer-readDocumentPointer-failure.feature

```diff
@@ -121,3 +121,27 @@ Feature: Producer Read Failure scenarios
       | issue_code        | VALIDATION_ERROR                                          |
       | issue_description | A parameter or value has resulted in a validation error   |
       | message           | Input is not composite of the form a-b: 8FW23\|1234567890 |
+
+  Scenario: Producer searches for another Producer's Document Pointer with the same ODS Code

```

Comment:
i mean this is right but I feel like the mention of suffix should exist in the name, like:
 `Producer under the same ODS code tries to access another producers document pointer`
 `Producers with the same ODS code but different suffix cant access another's document pointer` (this one feels too techy but i hope you get what I mean)

Reply:
Good point, will amend.